dgv/monitoring/hvd/task_functions.py

```diff
@@ -137,17 +107,24 @@ def markdown_item(row):
 
 def publish_mattermost(ti):
     filename = ti.xcom_pull(key="filename", task_ids="get_hvd")
-    goal = ti.xcom_pull(key="goal", task_ids="get_hvd")
-    # goal_list = ti.xcom_pull(key="goal_list", task_ids="get_hvd")
     minio_files = sorted(minio_open.get_files_from_prefix('hvd/', ignore_airflow_env=True))
     print(minio_files)
     if len(minio_files) == 1:
         return
+    all_hvd_names = set(get_table_as_df(

```

Comment:
I wonder if we should create a dedicated table with all HVD names? We're currently using an "archived" table that may get deleted at some point.